src/modules/displayfood.js

```diff
@@ -0,0 +1,192 @@
+import { foodApiObj, commentUrl, likeUrl } from './APIs.js';
+import Likes from './likes.js';
+
+import Comment from './comments.js';
+// import { modalDisplay } from './modules/commentpop.js';
+const main = document.querySelector('.main');
+
+const popup = document.querySelector('#modalDisplay');
+
+/* eslint-disable */
+export const display = async () => {
+/* eslint-enable */
```

Comment:
- [ ] Same issue as discussed above, please enable linters and try solving the issues involved.